File: examples/bukkit-java/src/main/java/com/example/ExamplePlugin.java
package com.example;

import com.squareup.moshi.Moshi;
import revxrsal.zapper.ZapperJavaPlugin;

public class ExamplePlugin extends ZapperJavaPlugin {

    @Override
    public void onEnable() {
        System.out.println(Moshi.class);
    }

    @Override
    public void onDisable() {
    }
}

File: examples/bukkit-kotlin/src/main/java/com/example/ExamplePlugin.kt
package com.example

import revxrsal.zapper.ZapperJavaPlugin

class ExamplePlugin : ZapperJavaPlugin() {

    companion object {
        init {
            println("Kotlin is already loaded!")
        }
    }

    override fun onEnable() {
        println("Hello from Kotlin!")
    }

    override fun onDisable() {
    }
}
